bitnami/schema-registry/values.yaml

```diff
@@ -639,6 +639,15 @@ kafka:
   ## @param kafka.replicaCount Number of Kafka brokers
   ##
   replicaCount: 1
+  ## @param kafka.auth.clientProtocol Authentication protocol for communications with clients. Allowed protocols: `plaintext`, `tls`, `mtls`, `sasl` and `sasl_tls`
+  ##
+  auth:
+    clientProtocol: "plaintext"
+  ## @param kafka.service.ports.client Kafka svc port for client connections
+  ##
+  service:
+    ports:
+      client: 9092
```

Comment:
Are these parameters added as a typo or is a conscious decision? I don't have any strong thoughts about this but could you please explain why you included them if done on purpose?

Reply:
Without these default values, the chart won't install, if used with `kafka.enabled: false` and external Kafka.
This was already discovered in the issue #16963 (which I added to this PR's description, for documentation purposes).
The problem is, these values are referenced e.g. in https://github.com/bitnami/charts/blob/main/bitnami/schema-registry/templates/statefulset.yaml#L5, but there is no check if kafka is enabled as a subchart, or external Kafka is used. 
So I had to fix it too, in order to test my changes. Do you have another suggestion how to fix it?

Reply:
You are right, I didn't notice the issue was on the upgrade phase and couldn't reproduce it the other day. In this case, I think this is the cleanest solution without having to rework any logic.